src/go/k8s/config/e2e-tests/manager.yaml

```diff
@@ -17,10 +17,10 @@ spec:
         - "--configurator-base-image=localhost/configurator"
         - "--configurator-tag=dev"
         - "--configurator-image-pull-policy=Never"
-        - "--allow-pvc-deletion=true"
         - "--superusers-prefix=__redpanda_system__"
         - "--log-level=trace"
         - "--unsafe-decommission-failed-brokers=true"
+        - "--unbind-pvcs-after=5s"
```

Comment:
If we needed to force unbind a PVC, I guess the route will be through k9s or kubectl by explicitly deleting the PVC?